.github/workflows/main.yml

```diff
@@ -144,3 +144,14 @@ jobs:
           signoff: true
           body: Add v${{ github.event.inputs.tag }} to community operators
           committer: quay-devel <quay-devel@redhat.com>
+
+      - name: Create Pull Request k8s-operatorhub
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.PAT }}
+          commit-message: Add v${{ github.event.inputs.tag }} to community operators
+          author: quay-devel <quay-devel@redhat.com>
+          push-to-fork: quay/community-operators 
```

Comment:
Is this the correct fork name? This has to be pre-configured

Reply:
Should be https://github.com/quay/community-operators